### PR TITLE
Feature revamp homepage

### PR DESCRIPTION
**Pull request by Ashwin-op**
- Home page events section

### CODE REVIEW

File: projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {{registrations}} Registered
+  </div>
+  <div [routerLink]="['/communities', community.slug, 'events', event.slug]" class="home-events-image clickable">
+    <img *ngIf="event.header_image_path" alt="{{event.name}}" src="{{event.header_image_path}}">
+    <div *ngIf="!event.header_image_path">
+      <app-skeleton-text [rows]="1"></app-skeleton-text>
+    </div>
+  </div>
+  <div class="home-events-date">
+    {{event.start_time | date:'MMM d'}}
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html]: Should we use moment and display this date/time in the current user's local clock time?

File: projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  </div>
+  <div class="home-events-details">
+    <p [routerLink]="['/communities', community.slug, 'events', event.slug]" class="home-events-name clickable">
+      {{event.name | truncatetext:35}}
+    </p>
+    <div class="home-events-host">
+      <nb-user [name]="community.name" [picture]="community.logo_path" [routerLink]="['/communities', community.slug]"
+               [showTitle]="false"></nb-user>
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html]: Let's use our own image tag, nb-user fails to position images properly sometimes

File: projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.scss [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    width: 56px;
+    top: 120px;
+    right: 12px;
+    border-radius: $border-8;
+    background-color: $neon-blue;
+    color: $white;
+    padding: $space-8 $space-12;
+    font-size: 16px;
+    font-family: $primary-font-family;
+    font-weight: 500;
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.scss]: Let's get standard font-weights from Apoorva, she said she's added them to figma, can you please figure this one out

**Review by arpansac — CHANGES_REQUESTED** [review]
Some minor changes. Perfect work :)

File: projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.scss (hunk 3 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.scss]: Done!

File: projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html (hunk 2 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html]: Done!

File: projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html (hunk 1 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html]: Sure, done!

**Pull request merged**